### DISCUSSION TITLE
How can I control the Crazyfile

### DISCUSSION THREAD

**Discussion by SUJUSTDO** [bitcraze/discussions#1647] [Q&A] [answered]
Hi, I'm trying to control the flight of the Crazyflie using the output from the AI-deck. Specifically, I want to manage the drone's forward and backward movement, as well as its left and right steering. Is this feasible? If so, which variables should I adjust, such as the forward/backward speed or steering angle (or speed)? Lastly, how do I set the appropriate magnitude for the forward or steering speed values? I apologize for the numerous questions, and I look forward to your response.

**Comment by Bananazooka** [comment]
Hi,  first of all, do you have any other decks than the AI deck? Or are you trying this with the AI deck only?

**Reply by SUJUSTDO** [reply]
Hi，I have the Flow Deck 2.0 and of course it would be better if it only relied on the AI deck.

**Reply by gemenerik** [reply, marked as answer]
Check out #1275 for advice on how to send data from GAP8 to STM32.
Control the Crazyflie from an STM32 firmware app, one example of where the drone is controller from the app layer is [the wall following example ](https://github.com/bitcraze/crazyflie-firmware/blob/master/examples/demos/app_wall_following_demo/src/wall_following.c).

**Reply by SUJUSTDO** [reply]
Thank you for your answer, I would like to be sure about one thing, can I use the reverse speed (-x) to control the drone to go backwards?

**Reply by gemenerik** [reply]
Yep!
